### ISSUE TITLE
Datagrid examples don't work

### ISSUE THREAD

**Issue by sidviny** [posit-dev/py-shiny#1364]
Hi, 

I'm running an app that was created with "shiny create" -> "dashboard example".  That app works seems to start up nicely, but it does not show the datagrid.  

The reason behind this, is mostly this: there are errors in the Javascript console (Google Chrome) telling that the mime type is not correct.  I attached a screenshot.

![Capture](https://github.com/posit-dev/py-shiny/assets/107187292/ba30bda3-1e2d-44ad-9d73-f4ef0501716c)

I have tested with both Shiny core as Shiny express.  Both samples of the datagrid (core / express) seem to have the same problem.

**Comment by cpsievert** [comment]
I'm unable to replicate this -- would you mind including another screenshot of what you see when going to the "Network" tab and select the "dataframe.js" file? I'm mainly interested in the response headers. This is what I see:

<img width="1032" alt="Screenshot 2024-05-08 at 10 53 23 AM" src="https://github.com/posit-dev/py-shiny/assets/1365941/01817c2c-1bb8-4182-9e1a-88c4aef0d926">

**Comment by sidviny** [comment]
Hi, sorry for the late response.  Long weekend here in Belgium ...

![image](https://github.com/posit-dev/py-shiny/assets/107187292/5d60a25b-17da-4dca-ba7c-b29618c43308)

![image](https://github.com/posit-dev/py-shiny/assets/107187292/73043fce-3c6d-4e2e-a8a6-9378ba102c98)

![image](https://github.com/posit-dev/py-shiny/assets/107187292/6e0360e3-c5fe-4cd0-a700-c9b6598148ab)

**Comment by sidviny** [comment]
Further information:

I'm running on Windows, with a conda environment created as local admin because I need to write in c:\ProgramData\Miniconda3, which is by default protected by our company.

Strangely, I have also a WSL environment on the same machine.  When I run this in a conda environment that is rebuild with the same packages, and I run it from WSL, I do get to see the data-view.

Here's the same info as above, but when started from WSL

![image](https://github.com/posit-dev/py-shiny/assets/107187292/beb24d06-1b32-40ed-a448-efdc0adca168)

![image](https://github.com/posit-dev/py-shiny/assets/107187292/0820cf1a-38cd-4f6f-95d2-4aafba9871d9)

**Comment by cpsievert** [comment]
Just curious, is this still a problem @sidviny?

**Comment by sidviny** [comment]
No, I just created this again and now everything is fine with this sample application

**Comment by cpsievert** [comment]
Great, thanks for checking!
